src/main/java/seedu/address/model/person/Person.java

```diff
@@ -93,12 +123,14 @@ public boolean equals(Object other) {
                 && otherPerson.getEmail().equals(getEmail())
                 && otherPerson.getAddress().equals(getAddress())
                 && otherPerson.getTags().equals(getTags());
+//                && otherPerson.getPrevDateMet().equals(getPrevDateMet())
+//                && otherPerson.getInfo().equals(getInfo());
```

Comment:
May I know why this is commented out?

Reply:
Will be discussed regarding the date data as I think it caused some check issues.